src/Controllers/DatabaseController.js

```diff
@@ -588,6 +602,9 @@ class DatabaseController {
               'Object not found.'
             );
           }
+          if (validateOnly) {
+            return result;
```

Comment:
what will `result` be in a `validateOnly` case?

Reply:
In both cases (create and update), it is returning the original object.